specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json

```diff
@@ -0,0 +1,345 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2.0",
+    "title": "Anomaly Finder",
+    "description": "The Anomaly Detection Service detects anomalies automatically in time series data. It supports two functionalities, one is for detecting the whole series with model trained by the timeseries, another is detecting last point with model trained by points before. By using this service, business customers can discover incidents and establish a logic flow for root cause analysis. Ensure online service quality is one of the main reasons we develop this service. Improving anomaly detection service to provide precise results will always be fundamental work of our team."
+  },
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    }
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{Endpoint}/anomalyfinder/v2.0",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "paths": {
+    "/timeseries/entire/detect": {
+      "post": {
+        "summary": "Find anomalies for the entire series in batch.",
+        "description": "The operation will generate a model using the entire series, each point will be detected with the same model. In this method, points before and after a certain point will be used to determine whether it's an anomaly. The entire detection can give user an overall status of the time series.",
+        "operationId": "post-timeseries-entire-detect",
```

Comment:
operationId should be {OperationGroupName}_{OperationName}

in SDK, this API will be mapped to
`AnomalyFinderClient.{OperationGroupName}Operations.{OperationName}()`

So we need to be careful about these names.

Reply:
done